lib/evss/base_service.rb

```diff
@@ -18,7 +18,7 @@ def get(url)
     def post(url, body)
       conn.post url do |req|
         req.headers['Content-Type'] = 'application/json'
-        req.body = body
+        req.body = body.to_json
```

Comment:
In my PR, I change this to

```ruby
    def post(url, body = nil, headers = { 'Content-Type' => 'application/json' }, &block)
      conn.post(url, body, headers, &block)
    end
```

Maybe what we could do is have this be the default block, but allow other blocks to be passed in, so that the uploaded doesn't end up having to work around the `.to_json` call.